spectrum-offchain-lm/src/executor.rs

```diff
@@ -152,7 +152,7 @@ where
                 let bundle_ids = ord.get::<Vec<BundleId>>();
                 let bundle_resolver = Arc::clone(&self.bundle_repo);
                 let bundles = stream::iter(bundle_ids.iter())
-                    .scan((), move |_, bundle_id| {
+                    .filter_map(move |bundle_id| {
```

Comment:
The `scan` call will terminate as soon as a box id is found to not be associated with a bundle. This can happen for a redeem order that was executed after a compound TX is formed locally by the bot. We should keep inspecting the other box ids, hence the need to replace with `filter_map`.